templates/myid.j2

```diff
@@ -1 +1,9 @@
+{% if install_mode == "standalone" %}
 {{ zoo_id }}
+{% elif install_mode == "cluster" %}
+{% for server in play_hosts %}
```

Comment:
I also don't like the fact we are using the implicit ordering of the host in the play to determine the id. Even though the hosts may appear in a specific order in the inventory file, the ordering they will be executed is not guarantee, it changes from one execution to another. Also if you execute a playbook by limiting the a single host, you will end up assigning an id of 1 to more than one ZK node, which will break the system.